lib/assets/components/explore_tiles.dart

```diff
@@ -260,10 +263,17 @@ abstract class ExploreResourceTile extends ExploreTile {
                       textScaler: const TextScaler.linear(.8),
                     ),
                     Expanded(child: Container()),
-                    if (isCompleted != null)
-                      _ExploreTileCheckmark(
-                        completed: isCompleted!,
-                      ),
+                    BlocBuilder<UserProgressBloc, UserProgressState>(
+                      builder: (context, state) {
+                        final isComplete = getIsComplete(state);
+                        if (isComplete) {
+                          return _ExploreTileCheckmark(
+                            completed: isComplete,
```

Comment:
Yeah we had isCompleted as nullable before I think maybe because if the user was not logged in we treated it as null (no completion data) vs false (user has not competed this). 

For now we only use 2 states so ive just gone with a required bool